NAMESPACE

```diff
@@ -1,21 +1,6 @@
-# Generated by roxygen2 (4.0.2): do not edit by hand
+# Generated by roxygen2 (4.1.0.9000): do not edit by hand
 
-export(date_parser)
-export(discretizer)
-export(drop_percent_missing)
-export(drop_single_value_variables)
-export(drop_variables)
-export(imputer)
-export(orderer)
-export(paramatch)
-export(parse_datetime)
-export(remove_outliers)
-export(renamer)
-export(select_rows)
-export(select_variables)
-export(sure_independence_screen)
-export(truncator)
-export(value_replacer)
+export()
```

Comment:
I ran document() and it made these changes.  Maybe the newer version does not require explicit function names?